scripts/onboarding/aks/onboarding-msi-azure-policy/azure-policy.rules.json

```diff
@@ -0,0 +1,323 @@
+{
+	"if": {
+	  "field": "type",
+	  "equals": "Microsoft.ContainerService/managedClusters"
+	},
+	"then": {
+	  "effect": "deployIfNotExists",
+	  "details": {
+		"type": "Microsoft.ContainerService/managedClusters",
+		"name": "[field('name')]",
+		"roleDefinitionIds": [
+			"/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c"
```

Comment:
you didnt find low privilege built-in roles than contributor?

Reply:
after adding those 3 roles https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/data-collection-rule-overview#permissions, the deployment still has error 
![image](https://github.com/microsoft/Docker-Provider/assets/25573844/0fc008c9-f306-47aa-a878-3ccf84295eb0)